flash_attn/flash_attn_triton_kernel_amd.py

```diff
@@ -373,6 +384,16 @@ def attn_fwd(Q, K, V, bias, sm_scale, L, Out, stride_qz, stride_qh, stride_qm, s
         seqlen_q = MAX_SEQLENS_Q
         seqlen_k = MAX_SEQLENS_K
 
+        
+    if USE_CACHE_SEQLENS:
```

Comment:
So what happens if `VARLEN and USE_CACHE_SEQLENS is True` ?

Reply:
This change has been moved to the decode kernel which didnot have varlen.